xrpl/transaction/amm_vote.go

```diff
@@ -1,14 +1,79 @@
 package transaction
 
+import (
+	"errors"
+
+	"github.com/Peersyst/xrpl-go/xrpl/ledger-entry-types"
+)
+
+// Vote on the trading fee for an Automated Market Maker instance. Up to 8 accounts can vote in proportion to the amount of the AMM's LP Tokens they hold.
+// Each new vote re-calculates the AMM's trading fee based on a weighted average of the votes.
+//
+// Example:
+//
+// ```json
+//
+//	{
+//	    "Account" : "rJVUeRqDFNs2xqA7ncVE6ZoAhPUoaJJSQm",
+//	    "Asset" : {
+//	        "currency" : "XRP"
+//	    },
+//	    "Asset2" : {
+//	        "currency" : "TST",
+//	        "issuer" : "rP9jPyP5kyvFRb6ZiRghAGw5u8SGAmU4bd"
+//	    },
+//	    "Fee" : "10",
+//	    "Flags" : 2147483648,
+//	    "Sequence" : 8,
+//	    "TradingFee" : 600,
+//	    "TransactionType" : "AMMVote"
+//	}
+//
```

Comment:
Remove comment empty lines

Reply:
This empty line is added by the Golang formatter in VS Code, it should be fine to leave it like this no?